__ti13bad_exception:
	...

